frontend/src/pages/home/videos/sections/FundingSection.tsx

```diff
@@ -0,0 +1,126 @@
+
+  return (
+    <>
+      <SectionTitle title={t('fundingSection.supportUs')} />
+      <Box display="flex" justifyContent="center" mb={6}>
+        <Box sx={{ width: { lg: '44%', xl: '44%' } }}>
+          <Typography variant="h3" textAlign="center" letterSpacing="0.8px">
+            {t('fundingSection.byFinancingTournesolYou')}
+          </Typography>
+        </Box>
+      </Box>
+      <Grid2 container spacing={4} justifyContent="center">
+        <Grid2 sm={12} md={4} lg={4} xl={3}>
+          <Paper elevation={0}>
+            <Box
+              p={2}
+              height="110px"
+              display="flex"
+              justifyContent="center"
+              alignItems="center"
+              color="#fff"
+              bgcolor="#1282B2"
+              borderRadius={1}
+            >
+              <Typography
+                m={0}
+                variant="h4"
+                textAlign="center"
+                sx={{
+                  '& span': {
+                    color: theme.palette.primary.main,
+                  },
+                }}
+              >
+                <Trans i18nKey="fundingSection.guaranteeTheProjectsIndependence">
+                  <span>guarantee</span> the project&apos;s independance
+                </Trans>
+              </Typography>
+            </Box>
+          </Paper>
+        </Grid2>
+        <Grid2 sm={12} md={4} lg={4} xl={3}>
+          <Paper elevation={0}>
+            <Box
+              p={2}
+              height="110px"
+              display="flex"
+              justifyContent="center"
+              alignItems="center"
+              color="#fff"
+              bgcolor="#1282B2"
+              borderRadius={1}
+            >
+              <Typography
+                m={0}
+                variant="h4"
+                textAlign="center"
+                sx={{
+                  '& span': {
+                    color: theme.palette.primary.main,
+                  },
+                }}
+              >
+                <Trans i18nKey="fundingSection.participateInTheCreationOfCommons">
+                  <span>participate</span> in the creation of commons
+                </Trans>
+              </Typography>
+            </Box>
+          </Paper>
+        </Grid2>
+        <Grid2 sm={12} md={4} lg={4} xl={3}>
+          <Paper elevation={0}>
+            <Box
+              p={2}
+              height="110px"
+              display="flex"
+              justifyContent="center"
+              alignItems="center"
+              color="#fff"
+              bgcolor="#1282B2"
+              borderRadius={1}
+            >
+              <Typography
+                m={0}
+                variant="h4"
+                textAlign="center"
+                sx={{
+                  '& span': {
+                    color: theme.palette.primary.main,
+                  },
+                }}
+              >
+                <span>agissez</span> pour
```

Comment:
I think you forget the translation for this one, or in fact you hard code the french version 😅

Reply:
Yes! In fact I didn't forget, I'm still thinking about what kind of impacting and motivating sentence we could use here.

Reply:
Fixed by https://github.com/tournesol-app/tournesol/pull/1291/commits/90fa16774465631576d1a8cbd3c9a29760b7f4bd.